src/Compilers/Core/Portable/Operations/ControlFlowGraphBuilder.cs

```diff
@@ -907,6 +907,65 @@ public override IOperation VisitWhileLoop(IWhileLoopOperation operation, int? ca
             return null;
         }
 
+        public override IOperation VisitVariableDeclarationGroup(IVariableDeclarationGroupOperation operation, int? captureIdForResult)
+        {
+            // We erase variable declarations from the control flow graph, as variable lifetime information is contained in a parallel data structure.
+            foreach (var declaration in operation.Declarations)
+            {
+                VisitVariableDeclaration(declaration, captureIdForResult);
+            }
+
+            return null;
+        }
+
+        public override IOperation VisitVariableDeclaration(IVariableDeclarationOperation operation, int? captureIdForResult)
+        {
+            foreach (var declarator in operation.Declarators)
```

Comment:
>var [](start = 21, length = 3)

Please no ```var```s. The type must be spelled out unless it is spelled out explicitly in the initializer, which is not present here.

Reply:
Fixed.

---
In reply to: [166775498](https://github.com/dotnet/roslyn/pull/24692#discussion_r166775498) [](ancestors = 166775498)